src/ddmd/traits.d

```diff
@@ -403,6 +404,50 @@ extern (C++) Expression semanticTraits(TraitsExp e, Scope* sc)
         return new IntegerExp(e.loc, false, Type.tbool);
     }
 
+    /**
+     Params: o: the AST-node to check.
+     Params: fdp: optionally a pointer to a function declararion,
+        which will get set o is a function declarartion.
+
+     Returns: a function-type if
+        o is a declarion of type  delegate, function,
+            or function pointer
+        or a variable of those types.
```

Comment:
When using periods, please do not put them in the middle of sentences. Replace it with a `,`